src/main/java/randoop/main/Minimize.java

```diff
@@ -57,12 +57,7 @@
 import plume.Options;
 import plume.TimeLimitProcess;
 import randoop.Globals;
-import randoop.output.ClassRenamingVisitor;
-import randoop.output.ClassTypeNameSimplifyVisitor;
-import randoop.output.ClassTypeVisitor;
-import randoop.output.FieldAccessTypeNameSimplifyVisitor;
-import randoop.output.MethodTypeNameSimplifyVisitor;
-import randoop.output.PrimitiveAndWrappedTypeVarNameCollector;
+import randoop.output.*;
```

Comment:
It's considered better Java style to use explicit imports (even though, if you are not using an IDE, wildcard imports can be easier to write).  So, please revert this change.

Reply:
Intellij frequently simplifies my imports when I press alt-enter to auto import a missing import. I will need to change that setting and revert this change.